pandas/conftest.py

```diff
@@ -0,0 +1,26 @@
+import pytest
+
+
+def pytest_addoption(parser):
```

Comment:
I don't like where it's at now either, but I think it's necessary. As [this note](http://docs.pytest.org/en/latest/writing_plugins.html#_pytest.hookspec.pytest_addoption) says:

>This function [pytest_addoption] should be implemented only in plugins or conftest.py files situated at the tests root directory due to how pytest discovers plugins during startup.

The other option, I suppose is to bundle a [setuptools entrypoint](http://docs.pytest.org/en/latest/writing_plugins.html#making-your-plugin-installable-by-others) so that our custom options show up for anyone who has pytest and pandas. We could prefix our `--run-slow`, etc with `--pandas-run-slow`, to avoid name clashes.

Do you have a preference between the two?